07-documenting_R.Rmd

```diff
@@ -98,10 +96,14 @@ Our dev guide workflow uses continuous integration and deployment so changes to
 
 ## Instructions for reporting the CRAN policy bugs -- discussion in slack (random channel)
 
+If you find some typo or problem on [CRAN](https://cran.r-project.org) after checking the problem or typo you can write a polite email to `cran-sysadmin@r-project.org` and one of the R-core members working with CRAN. You'll probably get a prompt reply about how the issue is going to be fixed.
```

Comment:
I think we might want to move this to a new chapter. There are various websites that the R Core team maintain, each with their own way to report issues (R FAQ, Windows FAQ, Mac FAQ, R project web-site, CRAN website). These are all outside the scope of Bugzilla. This might be a new section in the Bug reporting chapter (since we also document there how to report bugs in non-R-Core maintained packages), or a small chapter on "R project websites". Perhaps the latter is useful as there are also other sites on the r-project domain, like translation.r-project,org and contributor.r-project.org that could be good to document briefly somewhere.

Reply:
Would this also apply to the manuals?